plugins/woocommerce/src/Blocks/BlockTypes/BlockifiedProductDetails.php

```diff
@@ -263,13 +330,55 @@ private function has_accordion( $parsed_block ) {
 	}
 
 	/**
-	 * Get the frontend script handle for this block type.
+	 * Register a product details item using Block Hooks API.
 	 *
-	 * @see $this->register_block_type()
-	 * @param string $key Data to get, or default to everything.
-	 * @return array|string|null
+	 * @param string $title The title of the item.
+	 * @param string $content The content of the item.
+	 * @return void
 	 */
-	protected function get_block_type_script( $key = null ) {
-		return null;
+	private function register_hooked_blocks( $title, $content ) {
+		$slug = sanitize_title( $title );
```

Comment:
Nit: definitely not a blocker as it might not be an issue in practical terms. Would it be necessary to introduce a static variable to keep track of registered slugs. At the moment, there is a risk that the same virtual block type names could get duplicated which might lead to unknown consequences as we deal with filters registration.